apps/glusterfs/app_config.go

```diff
@@ -36,6 +36,7 @@ type GlusterFSConfig struct {
 	PreReqVolumeOptions  string `json:"pre_request_volume_options"`
 	PostReqVolumeOptions string `json:"post_request_volume_options"`
 	ZoneChecking         string `json:"zone_checking"`
+	maxVolumesPerCluster int    `json:"max_volumes_per_cluster"`
```

Comment:
Doesn't it need to be "MaxVolumesPerCluster" to make the JSON parsing happy? I think it's also needed if we expect to programmatically set the config as we do in some of the newer functional tests.

Reply:
done